README.md

```diff
@@ -56,8 +57,9 @@ For platforms with Autotools, MEGAcmd can be built and installed with:
     ./configure
     make
     make install
+    ldconfig
```

Comment:
That wouldn't be strictly necessary

Reply:
I had to use it on Ubuntu 16.04. Maybe it is not always necessary. Removed